.github/workflows/check_libxrpl.yml

```diff
@@ -0,0 +1,91 @@
+name: Check new libXRPL
+on:
+  repository_dispatch:
+    types: [check_libxrpl]
```

Comment:
Where is this type defined?

Reply:
It's not necessarily defined anywhere afaik. It's the same string that we pass from `rippled` CI though so it matches this particular flow using it.